### PR TITLE
feat: use i18nexus translations

### PR DESCRIPTION
**Pull request by danielgent**
Welcome to the dark side Micke

image

Same approach as I did on https://github.com/sesamyab/login2/pull/495

When can I get Julian to do the Italian translations?  On a serious note, I think the flow of an invited translator is the one we really want to try out

### CODE REVIEW

File: src/controllers/email.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { translate } from "../utils/i18n";
 import { Client, Env } from "../types";
 import sendEmail from "../services/email";
 import { getClientLogoPngGreyBg } from "../utils/clientLogos";
-import { en, sv } from "../locales";
+import en from "../../public/locales/en/default.json";
```
Comment (danielgent) [inline src/controllers/email.ts:6]: default location for i18next.  I'm sure the nexus-cli will have args to change this

File: src/utils/i18n.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { en, sv } from "../locales";
+import en from "../../public/locales/en/default.json";
+import sv from "../../public/locales/sv/default.json";
```
Comment (danielgent) [inline src/utils/i18n.ts:2]: see below, even though I've changed these to JSON, the line
`export type LabelIds = keyof typeof en | keyof typeof sv;`
gives us typing :exploding_head:  as use `LabelIds` as one of the params

*SO* when I copy-pasted the same login2 nexus key, typescript warned me :exploding_head:

File: public/locales/en/default.json
Comment (danielgent) [inline public/locales/en/default.json:1]: :smiling_face_with_tear: sadly there's no diff. If only Microsoft spent $50 billion on GitHub diff farms

File: package.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
     "test-with-coverage": "jest --config jest.config.js --coverage",
     "semantic-release": "semantic-release",
     "prepare": "husky install",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "pull-translations": "i18nexus pull -k EWI52U0e8j13AX31kLcS-A && prettier --write ./public/locales/**/*.json"
```
Comment (danielgent) [inline package.json:20]: Simplest approach copied from https://github.com/sesamyab/login2/pull/495

Comment (markusahlstrand) [reply package.json:20]: Cool!

**Review by markusahlstrand — APPROVED** [review]
Looks good!

**Pull request merged**